technology-adapter-api.md

```diff
@@ -26,6 +26,62 @@ Message:
 
 val - is error code, src - origin of the error.
 
+## Requesting status of a single device.
+
+An adapter has to support an API for requesting a device state.
+
+### Command
+
+Topic: `pt:j1/mt:cmd/rt:ad/rn:zigbee/ad:1`
+
+Message:
+
+```json
+{
+    "serv": "zigbee",
+    "type": "cmd.network.get_node",
+    "val_t": "int",
```

Comment:
It's address so string. Check out this:
https://github.com/futurehomeno/fimp-api/blob/master/thing-management.md#getting-inclusion-report

On adapter side a THING has only one identification and it is an THINGADDRESS.

Reply:
fixed